docs/product/components/inputs.html

```diff
@@ -10,7 +10,7 @@
 
     <div class="stacks-preview">
 {% highlight html %}
-<div class="d-flex gs4 gsy fd-column">
+<div class="d-flex gx4 fd-column">
```

Comment:
## Change requested

Since `<label>` elements are rendered as `display: table`, the gap property doesn't apply. Any `<label>` with parents that include a gap property need either to have a margin added or have their `display` property altered to honor `gap`.

Reply:
I cannot see any visual difference here. Can you have another look in the updated PR and tell me if this is still relevant? Thanks

Reply:
Ah, I was a little hasty in assuming the same issue was here. Backstop show some significant differences on this page and it looks to be due to `.s-input-message` being rendered with the `<p>` tag, which comes with margins that were previously overwritten with the `.gs4` class.

Reply:
👍